User_Setups/Setup73_M5Stack_Core2.h

```diff
@@ -0,0 +1,27 @@
+#define USER_SETUP_ID 73
+
+#define ILI9341_DRIVER
```

Comment:
Actually I'd like to use ILI9342, since this is a landscape device.  But it would require adjusting the mirror-image correction in ILI9341_Rotation.h.  Something like this:

```c++
    case 0:
#ifdef M5STACK
    #ifdef ILI9342_DRIVER
      writedata(TFT_MAD_COLOR_ORDER);
    #else
      writedata(TFT_MAD_MY | TFT_MAD_MV | TFT_MAD_COLOR_ORDER);
    #endif
#else
      writedata(TFT_MAD_MX | TFT_MAD_COLOR_ORDER);
#endif
      // ... (and similar for other rotation cases)
```

Happy to add that refinement, though things are usable as-is with ILI9341 and `setRotation(1)`.

Reply:
Considering it more, using the landscape driver would cause compatibility annoyance because setRotation(0) would have a different meaning between Core Basic (which also has landscape display), Core2 on M5Stack's TFT_eSPI fork, and Core2 using the stock TFT_eSPI config.  So I'll leave it as is.